src/components/ui/skeleton/ExplorePostItemSkeleton.tsx

```
@@ -0,0 +1,122 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { SkeletonBase } from './animations';
+import { colors, radius, fontSizes } from '@/styles/theme';
```

ESLint

[error] 'fontSizes' is defined but never used.

(@typescript-eslint/no-unused-vars)